AirCasting/Map/PathPoint.swift

```diff
@@ -19,6 +19,7 @@ struct PathPoint: Equatable {
     let location: CLLocationCoordinate2D
     let measurementTime: Date
     var measurement: Double = 20
+    #warning("Do something with hard coded measurement")
```

Comment:
plase adhere to our new warning standards (see readme) :)

Reply:
I need to say that I cannot see it on readMe 😬, should I add FIXME or smth else?

Reply:
> I need to say that I cannot see it on readMe 😬, should I add FIXME or smth else?

My bad 😅
README is updated now

Reply:
Thanks 🚀